Specs/DateTime/German/DatePeriodExtractor.json

```diff
@@ -1,12 +1,12 @@
 [
   {
-    "Input": "Ich werde diesen Januar nicht hier sein.",
+    "Input": "Diesen Januar werde nicht hier sein.",
```

Comment:
Same.

Reply:
I don't know why this was removed. I no have both forms in the spec and fixed a mistake in the new sentence.